components/imageproc/src/ops.rs

```diff
@@ -46,12 +53,21 @@ impl ResizeOperation {
             _ => return Err(anyhow!("Invalid image resize operation: {}", op)),
         };
 
+        let filter = match filter {
+            "lanczos3" => FilterType::Lanczos3,
+            "nearest" => FilterType::Nearest,
+            "triangle" => FilterType::Triangle,
+            "catmullrom" => FilterType::CatmullRom,
+            "gaussian" => FilterType::Gaussian,
+            _ => return Err(anyhow!("Invalid filter type: {}", filter)),
+        };
```

Comment:
Can we convert directly to the filter type from the image crate and skip our own?

Reply:
I went with a custom type because FilterType from the image crate is not `Hash`, so it breaks downstream functionality when the processed image paths are generated.

Reply:
I've just made a PR to impl it, let's see if it works

Reply:
i think it's released in the most recent version